Update compliance.rst (#3823)
mattermost/docs: source/overview/compliance.rst

FAQ
----

To be compliant with GDPR, do I need to remove message contents of email notifications?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Based on our interpretation of GDPR, it is not required to hide message contents in email notifications to remain compliant for the following reasons:

1. Every user has the ability to disable email notifications in their Account Settings. Therefore, every user has the ultimate control over whether or not they want information sent via email. This option aligns with most other products, but we will follow updates on interpretations of GDPR closely to see if we need to make changes in this area.
2. Mattermost offers :ref:`TLS encryption <email-tls>` to protect communication between the Mattermost server and the SMTP email server.
3. For those who are uncertain if the first two points cover GDPR compliance, we offer the ability to :ref:`disable notifications completely <email-notification-config>` on your Mattermost server. To use Mattermost in production with no email notifications, you also need to :ref:`disable a "preview mode" notice banner <email-preview-mode-banner-config>`.

> What information is shared when I click Contact us on the Mattermost Admin Advisor?

Reviewer: ```suggestion
What information is shared when I click Contact Us on a Mattermost Admin Advisor notification?
```

> What information is shared when I click Contact Us on a Mattermost Admin Advisor notification?

Author: ```suggestion
What information is shared when I click **Contact us** on a Mattermost Admin Advisor notification?
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> When you contact Mattermost using a **Contact Us** button, website form, or other method and supply your email address and contact details, that information is sent to us directly and used to contact you as requested.

Reviewer: For the "Contact Support" button for admin advisor notifications, the admin doesn't supply the email address, it's automatically sent to us.

> When you contact Mattermost using a **Contact Us** button, website form, or other method and supply your email address and contact details, that information is sent to us directly and used to contact you as requested.

Author: ```suggestion
When you contact Mattermost using a **Contact Us** button, website form, or other method and supply your email address and contact details, that information is sent to us directly and used to contact you as requested. When you contact us from within Mattermost, the contact details associated with your Mattermost instance are automatically appended and sent to us.
```

Reviewer: Let's make it very clear what this includes (e.g. email address). I also wonder if we should reference the admin advisor notifications specifically. I'll defer to Sadur.

> Clicking the **Contact Us** button shown in the Mattermost Admin Advisor will send some information to us. This may include the email address and name associated with your Mattermost account as well as the number of registered users on your system, the site URL, and a Mattermost diagnostic server ID number. This information is used to contact you as requested and to help us better understand your needs.

Author: ```suggestion
Clicking **Contact us** in the Mattermost Admin Advisor will send some information to us. This may include the email address and name associated with your Mattermost account as well as the number of registered users on your system, the site URL, and a Mattermost diagnostic server ID number. This information is used to contact you as requested and to help us better understand your needs.
```
